src/storage/fs/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './NodeFileSystem'
+export * from './FileSystem'
+export * from './ReactNativeFileSystem'
```

Comment:
We can't export this from the framework root as that means you automatically import `fs` if you import anything from the framework. `fs` is not available in React Native